docs/user_guide/ug_general_concepts.md

```diff
@@ -1 +1,42 @@
-TODO
\ No newline at end of file
+# General concepts
+
+MechaPhlowers is a library specialized to perform mechanical and geometrical calculus for powerlines. The models are designed to be the simplest and most accurate possible : when possible, calculus are performed in 2D planes.
+
+The granularity is important : the line is composed by several section. For each section, you have several cables. Each cable can be divided into spans, which is the lowest scale of calculus.
+
+[scheme granularity]
+Câble : Une phase peut être composée d’un unique câble ou d’un faisceau de câbles. Dans la suite on parle de câble de manière générique : phase constituée d'un câble simple, phase constituée d'un faisceau de câble, ou câble de garde.
+
+The algorithms are switching between the different levels during the resolution
```

Comment:
Je ne comprends pas trop cette phrase. Cela veut dire qu'on passe du câble, à la portée, au canton durant les calculs ?
Et si c'est bien cela, c'est vraiment utile de le dire ?

Reply:
Sentence is deleted